src/dlsym_interface.c

```diff
@@ -53,12 +53,12 @@ static void dlsym_interfaces_get_path(void)
 {
     if (!(libapduinterface_path = getenv("APDU_INTERFACE")))
     {
-        libapduinterface_path = "." DIRSEP "libapduinterface_pcsc." INTERFACELIB_POSTFIX;
+        libapduinterface_path = "libapduinterface_pcsc." INTERFACELIB_POSTFIX;
     }
 
     if (!(libhttpinterface_path = getenv("HTTP_INTERFACE")))
     {
-        libhttpinterface_path = "." DIRSEP "libhttpinterface_curl." INTERFACELIB_POSTFIX;
+        libhttpinterface_path = "libhttpinterface_curl." INTERFACELIB_POSTFIX;
```

Comment:
Did you test this change on Windows? I remember it might not be working on my Windows machine

Reply:
And you should remove the DIRSEP too

Reply:
Done. Remove upstream strange LOAD_WITH_ALTERED_SEARCH_PATH and avoid UB.
https://github.com/dlfcn-win32/dlfcn-win32/issues/104